docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -84,11 +82,11 @@ Now you have retrieved the relevant documentation, you can package content with
 
 The values from the CPIX documents will need to be copied and pasted to the corresponding section in the examples below.
 
-For more information, please refer to the Shaka Packager Documentation which can be found at the following link - [Shaka Packager Documentation](https://google.github.io/shaka-packager/html/)
+For more information, please refer to the documentation from [Shaka Packager](https://google.github.io/shaka-packager/html/).
```

Comment:
```suggestion
For more information, please refer to the documentation from Google on [Shaka Packager](https://google.github.io/shaka-packager/html/).
```
Think this makes more sense as the docs come from Google and Shaka Packager is the product.

Reply:
Actioned.